UnitsNet/CustomCode/Wrappers/Pressure.Wrapper.cs

```diff
@@ -0,0 +1,129 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnitsNet.Units;
+using UnitsNet.CustomCode.Units;
+
+namespace UnitsNet.CustomCode.Wrappers
+{
+    /// <summary>
+    /// From gratestas in #422
+    /// Pressure is a state function, for its measurement depends on environmental factors such as ambient pressure, elevation above sea level and local weather conditions. For two persons located in the different environment to speak of the same pressure, one must relate it to a reference. There are three basis reference: absolute, gauge, vacuum
+    ///    Absolute is zero-referenced to the total vacuum.
+    ///    Gauge refers to a level of the local atmospheric pressure.
+    ///    Vacuum is the negative of the gauge.
+    ///Therefore, to obtain consistent and qualitative data, the reference measurement is crucial.
+    /// </summary>
+    public struct ReferencePressure
+    {
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="pressure"></param>
+        /// <param name="reference"></param>
+        public ReferencePressure(Pressure pressure, PressureReference reference)
+        {
+            _reference = reference;
+            Pressure = pressure;
+        }
+
+        /// <summary>
+        /// ctor using BaseReference of absolute to assign default _reference
+        /// </summary>
+        /// <param name="pressure"></param>
+        public ReferencePressure(Pressure pressure)
+        {
+            _reference = BaseReference;
+            Pressure = pressure;
+        }
+
+        /// <summary>
+        ///     The public repersentation of the measured reference this quantity was constructed with.
+        /// </summary>
+        public PressureReference Reference => _reference.GetValueOrDefault(BaseReference);
+
+        /// <summary>
+        ///     The measured reference this quantity was constructed with.
+        /// </summary>
+        private readonly PressureReference? _reference;
```

Comment:
Does this need to be nullable? It seems to me it's always getting a value assigned in the ctors. I feel it is more natural to assign it `BaseReference` in the ctor that does not take a parameter, instead of doing `GetValueOrDefault()` in the getter property.

Reply:
Refactored as non-nullable property